NEWS

```diff
@@ -1,4 +1,8 @@
 * Version (not released, yet)
+ - m-kress:
+ -- do not substitute trailing control characters of a log line, but eliminate them completelly
```

Comment:
It might be good to mention that this breaks filters relying on the trailing underscore, even though most users never read these files.